forge-ai/src/main/java/forge/ai/ComputerUtil.java

```diff
@@ -433,12 +427,39 @@ public static Card getCardPreference(final Player ai, final Card activate, final
                 final CardCollection sacMeList = CardLists.filter(typeList, new Predicate<Card>() {
                     @Override
                     public boolean apply(final Card c) {
-                        return c.hasSVar("SacMe") && Integer.parseInt(c.getSVar("SacMe")) == priority;
+                        return (c.hasSVar("SacMe") && Integer.parseInt(c.getSVar("SacMe")) == priority)
+                                || (priority == 1 && shouldSacrificeThreatenedCard(ai, c, sa));
                     }
                 });
                 if (!sacMeList.isEmpty()) {
                     CardLists.shuffle(sacMeList);
-                    return sacMeList.get(0);
+                    return sacMeList.getFirst();
+                } else {
+                    // empty sacMeList, so get some viable average preference if the option is enabled
+                    AiController aic = ((PlayerControllerAi) ai.getController()).getAi();
```

Comment:
@tool4ever @Agetian seems the classcast and maybe others needs checking if the controller is human?

Reply:
Fixed, segments like that only need to run for the AI decisions anyway, so the .isAI check is mandatory there. Good and timely catch, thanks! :)